generators/app/templates/src/app/components/Routes/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { BrowserRouter as Router, Switch } from 'react-router-dom';
+
+import { useSelector } from '~app/context';
+
+import Suspense from '../Suspense';
+
+import { ROUTES } from './constants';
+import RouteItem from './components/RouteItem';
+import styles from './styles.module.scss';
+
+function AppRoutes() {
+  const user = useSelector(state => state.user);
+
+  return (
+    <Router>
+      <div className={styles.container}>
```

Comment:
I didn't put this here (I just changed it to TS). Nevertheless, I think it depends on how you look at it. It is not mandatory, but it might be good to have a container for the whole app with the height set to 100vh (or whatever size it should be) so we don't have to do that in every screen's container.